packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/challengeintegrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="common-image-format">
+
+![Sequence diagram showing Google challenge summary](/img/authenticators/java-authenticators-google-challenge-summary.png)
```

Comment:
Steps 5 onwards in the flow diagram need to match the steps in the text or the text needs to match the steps in the diagram